### DISCUSSION TITLE
Custom brute force template message

### DISCUSSION THREAD

**Discussion by henrique-arthur** [keycloak/keycloak#32523] [Q&A]
Hi everyone, is there any way to send a custom message when a login account is temporarily blocked by brute force attack?
I searched through the keycloak docs and found that it treats this as an "Invalid username or password", but I want to print a custom message.

What I mean is how do I know when a brute force error occurs and how to display a custom message for it?

**Comment by tonythomas1** [comment]
Hey, was looking for the same thing, have you found a solution ?
